packages/styles/src/themes/ds3.ts

```diff
@@ -1317,6 +1317,24 @@ const ds3 = makeTheme((theme) => ({
         },
       },
     },
+    HvWarningText: {
```

Comment:
Sorting these (roughly) alphabetically helps reducing diff conflicts (and those with OCD too 😅)

Reply:
Sorted!